packages/connection-form/src/hooks/use-connect-form-settings.tsx

```diff
@@ -33,7 +33,7 @@ const defaultSettings = {
   showKerberosAuth: true,
   showCSFLE: true,
   showProxySettings: true,
-  saveAndConnectLabel: 'Connect',
+  saveAndConnectLabel: 'Save & Connect',
```

Comment:
This is basically the magic that means this PR shouldn't break vscode.

Reply:
just fyi, VSCode provides it's own settings, it isn't using this hook. in case that's what you meant by this comment, not sure if I got it right. https://github.com/mongodb-js/vscode/blob/bfa57d4ce7b057fadb9fcec95327826f5d0dbb6b/src/views/webview-app/connection-form.tsx#L103

Reply:
That's what I meant - it uses the setting too. The setting is arguably redundant now, though: both apps set it to the same.